backend/src/main/java/com/carffeine/carffeine/service/ScrapperService.java

```diff
@@ -27,11 +28,17 @@ public class ScrapperService {
     private static final String DATA_TYPE = "JSON";
     private final RestTemplate restTemplate;
     private final ChargeStationRepository chargeStationRepository;
+
+    private final Random random = new Random();
```

Comment:
```suggestion
     private final ThreadLocalRandom threadLocalRandom= ThreadLocalRandom.current();
```

참고
https://velog.io/@sojukang/Random-%EB%8C%80%EC%8B%A0-ThreadLocalRandom%EC%9D%84-%EC%8D%A8%EC%95%BC-%ED%95%98%EB%8A%94-%EC%9D%B4%EC%9C%A0

Reply:
Random이 완전한 랜덤이 아니었다니 신기하네요.
새로운 기능 감사합니다~